backend/fms_core/templates.py

```diff
@@ -631,7 +632,7 @@
   "user prefill info": {
       "QC Date (YYYY-MM-DD)": "date",
       "Volume Used (uL)": "number",
-      "Quality Instrument": "qc-instrument",
+      "Quality Instrument": SAMPLE_QC_QUALITY_INSTRUMENTS,
       "Quality Flag": VALID_QC_FLAG_CHOICES,
       "Quantity Instrument": "qc-instrument",
```

Comment:
We should replace that list too. I think we will just get this request later otherwise (using the sample QC template quantity instrument list as a reference).